crates/update/src/cli/link.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::PathBuf;
+
+use anyhow::Context;
+use spacetimedb_paths::cli::BinDir;
+use spacetimedb_paths::SpacetimePaths;
+
+/// Set a local installation of SpacetimeDB as a custom version.
+#[derive(clap::Args)]
+pub(super) struct Link {
```

Comment:
I think this is new since my last review - why did we add this in here? It does seem convenient but it's whole extra bit of surface area that I think we could add in a backwards-compatible way later. Was it part of some spec somewhere or did Tyler request it or something?

Reply:
This wasn't spec'd, and I should've left it as a separate commit, sorry, but this felt important for dog-food QA testing - letting us install `spacetime-update` globally but still use our local version for testing and stuff.

Reply:
Also, it's left hidden for a reason - this isn't meant to be a stable cli surface.

Reply:
Now they're as a separate commit.